app/lib/actions/katello/host/attach_subscriptions.rb

```diff
@@ -27,7 +27,7 @@ def plan(host, pools_with_quantities_params)
         end
 
         def finalize
-          ::Katello::Pool.where(:id => input[:pool_ids]).each(&:import_data)
+          ::Katello::Pool.where(:cp_id => input[:pool_ids]).map { |pool| pool.import_data(false) }
```

Comment:
this doesn't seem correct, its hard to tell, but it looks like the pool_ids in the input are the actual katello db ids?

Reply:
for remove below it looks like it is actually passing the cp_ids, so i could see how this is confusing!

Reply:
everything else looks fine and runs fine.  if you want to squash along with this fix.